content/docs/user-guide/how-to/untrack-data.md

```diff
@@ -0,0 +1,45 @@
+# How to Un-track Data
+
+There are situations where you may want to stop tracking data added previously.
+Let's see how it can be done using an example `data.csv` file.
```

Comment:
I generalized this one (it's still basically "undo `dvc add`) but expressed in more generic terms that we can expand on later e.g. with pipelines.

Reply:
it's not undo `dvc add` though ... files is still in cache in this case. We should run `gc` or remove file manually from cache to make undo `dvc add`. Or at least mention about this.

Reply:
okay, I see that it is mentioned